capstone-frontend/src/App.jsx

```diff
@@ -22,27 +21,45 @@ class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      searchQuery: ""
+      searchQuery: "",
+      bookLists: []
```

Comment:
nit: Add a trailing comma here (https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma). That will give you cleaner diffs if you add more properties to this object in the future.

Reply:
Ok